tools/operator/paths/paths.go

```diff
@@ -158,13 +158,17 @@ func PTEPath() string {
 	return path
 }
 
-//GetConnProfilePathForOrg --
```

Comment:
I am confused why you removed the "ForOrg" from the name. You are still only getting one path, doing this for exactly one orgName (whether the only one, or the first one).

Reply:
If an array of orgs are passed, then this function just returns directory path where all connection profiles present. If an org is passed, this returns file path. So thought GetConnProfilePath would be better suited than GetConnProfilePathForOrg.